docs/architecture.md

```
@@ -1,51 +1,67 @@
 ## ディレクトリ構造
+
 // TODO
 
 ## SuteraVRを構成する要素
+
 ### Syncer
+
 **役割**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)